changelog.d/20240321_205232_kavikumarceo_resolved_issue_7443.md

```diff
@@ -0,0 +1,4 @@
+### Added
+
+- Skipped the frames with all resolved issues
```

Comment:
I belive its not an `added a feature` but fix a bug, The filter should've been skipping issues that are hidden